src/main/resources/reference.conf

```diff
@@ -1139,11 +1139,14 @@ resourceTypes = {
       update_passport_identifier = {
         description = "Can update a datasnapshot's passport identifier (e.g. consent code)"
       }
+      export_snapshot = {
+        description = "Export-by-copy of a snapshot to a Terra workspace"
```

Comment:
Minor thing but should this just be export as opposed to export-by-copy? Would the user need this then too?

Reply:
Yeah, on second thought I like the more general description since this could be repurposed for other kinds of exports

Reply:
That's fair!  This was made more specific on my request, if memory serves.